### PR TITLE
Add unit tests for com.tmobile.cso.pacman.datashipper.util.Util

### PR DESCRIPTION
**Pull request by Braavos6**
I've analysed your codebase and noticed that `com.tmobile.cso.pacman.datashipper.util.Util` is not fully tested.
I've written some tests for the methods in this class with the help of [Diffblue Cover](https://www.diffblue.com/opensource).

Hopefully, these tests will help you detect any regressions caused by future code changes. If you would find it useful to have additional tests written for this repository, I would be more than happy to look at other classes that you consider important.

### CODE REVIEW

File: jobs/pacman-data-shipper/src/test/java/com/tmobile/cso/pacman/datashipper/util/UtilTest.java [hunk 1]
```diff
@@ -0,0 +1,70 @@
...
+        HashMap<String, String> hash = new HashMap<>();
+        hash.put("foo", "3");
+
+        Assert.assertEquals("3",
+                Util.concatenate(hash, new String[]{"foo"}, ","));
+    }
+
+    @Test
+    public void testParseJson() {
+        Assert.assertEquals(new HashMap<>(), Util.parseJson("{foo{bar:3;}}"));
```
Comment (freddytuxworth) [inline jobs/pacman-data-shipper/src/test/java/com/tmobile/cso/pacman/datashipper/util/UtilTest.java]: Add a test with valid JSON

File: jobs/pacman-data-shipper/src/test/java/com/tmobile/cso/pacman/datashipper/util/UtilTest.java [hunk 2]
```diff
@@ -0,0 +1,70 @@
...
+        Assert.assertTrue(Util.contains(hash2, hash1, new String[]{"foo"}));
+    }
+
+    @Test
+    public void testConcatenate() {
+        HashMap<String, String> hash = new HashMap<>();
+        hash.put("foo", "3");
+
+        Assert.assertEquals("3",
+                Util.concatenate(hash, new String[]{"foo"}, ","));
```
Comment (freddytuxworth) [inline jobs/pacman-data-shipper/src/test/java/com/tmobile/cso/pacman/datashipper/util/UtilTest.java:29]: After this assert, do another `hash.put` with other values and then another test

**Review by freddytuxworth — CHANGES_REQUESTED** [review]

File: jobs/pacman-data-shipper/src/test/java/com/tmobile/cso/pacman/datashipper/util/UtilTest.java (hunk 2 above)
Comment (Braavos6) [reply jobs/pacman-data-shipper/src/test/java/com/tmobile/cso/pacman/datashipper/util/UtilTest.java:29]: Done

File: jobs/pacman-data-shipper/src/test/java/com/tmobile/cso/pacman/datashipper/util/UtilTest.java (hunk 1 above)
Comment (Braavos6) [reply jobs/pacman-data-shipper/src/test/java/com/tmobile/cso/pacman/datashipper/util/UtilTest.java]: Done

**Review by freddytuxworth — APPROVED** [review]

**Review by freddytuxworth — APPROVED** [review]

**Review by JohnLBergqvist — APPROVED** [review]

**Pull request closed without merging**